packages/vscode-boxel-tools/src/file-system-provider.ts

```diff
@@ -380,15 +378,7 @@ export class RealmFS implements vscode.FileSystemProvider {
       }
       console.log('Response:', response);
 
-      const contentType = response.headers.get('Content-Type');
-      let content: Uint8Array;
-
-      if (contentType && contentType.includes('application/json')) {
-        const jsonData = await response.json();
-        content = new TextEncoder().encode(JSON.stringify(jsonData, null, 2));
```

Comment:
I removed this because it was pretty-printing JSON files instead of displaying their extra contents, so a file with `[1,2,3]` showed up as

```
[
  1,
  2,
  3
]
```